src/arrays/what-requests.js

```diff
@@ -7,22 +7,25 @@ const requests = [
     id: 1,
```

Comment:
well, i wasn't sure about removing or changing the ID since Rebecca uses this for her component and i didnt want to potentially mess up her code. @thinklikeadesigner what do you think?